src/utils/SSRRenderer.ts

```diff
@@ -8,7 +7,7 @@ export function SSRRenderer(jsx: JSX.Element, container?: HTMLElement) {
             container = document.createElement('div')
             document.body.appendChild(container)
         }
-        ReactDOM.hydrate(React.createElement(React.StrictMode, {}, jsx), container)
+        ReactDOM.createRoot(container).render(jsx)
```

Comment:
Is concurrent mode stable now?

Reply:
Not stable yet. But it's convenient to load async data meanwhile provide a good user experience by using concurrent mode.

We also need to familiar with the next version of react because Facebook will use it.

No, this pr is using the feature from concurrent mode so it can't be splitted into another pr.

Reply:
Depend on an experimental feature could be dangerous.
It's better to have a buffer period that the old laggy contact list coexists with this new contact list powered by React concurrent mode. We can provide the user with a toggle in the settings panel to control that feature (maybe default on).

Reply:
We have package version lock so if it is working now, it will working on the client device. The concurrent mode has developed for years and become more stable now.

@septs what's your opinion?